web-app/src/localplay/games/localplay.games.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Body, Controller, Post, Put } from '@nestjs/common';
```

Comment:
Why do you choose to not set localplay files at the top of the localplay directory ?

Reply:
Since the REST route I chose is `'localplay/games'` I figured this would be more coherent. Do you disagree?

Reply:
Since this is probably a temporary endpoint I think we can leave it !